package.json

```diff
@@ -167,6 +168,11 @@
                     "dark": "images/dark/add.svg"
                 }
             },
+            {
+                "command": "vscode-kafka.explorer.copy",
```

Comment:
I think we should rename `vscode-kafka.explorer.copy` to `vscode-kafka.explorer.copy-label` andupdate title with `Copy Label` because it's a copy of label. 

We could provide in an another PR a Copy feature which copy for instance a topic and we do Ctrl+V in an another cluster or the same cluster,it could open the create topic wizards with the copied topic.

It will be consistent with File explorer : 

![image](https://user-images.githubusercontent.com/1932211/104291327-b065a100-54bb-11eb-9660-26d947d5f97c.png)

And it should be nice too to group commands like File explorer does.